sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -42,23 +42,27 @@ private SetStatementAnalyzer() {
     }
 
     public static SetAnalyzedStatement analyze(SetStatement node, ParameterContext parameterContext) {
-        if (SetStatement.Scope.SESSION.equals(node.scope())) {
-            logger.debug("SET SESSION STATEMENT: {}", node.toString());
+        if (!SetStatement.Scope.GLOBAL.equals(node.scope())) {
+            logger.warn("SET STATEMENT WITH SESSION OR LOCAL WILL BE IGNORED: {}", node);
```

Comment:
dispatch in the beginning between global and non global, since there can also be only one assignement in local,session too.